engine/src/main/java/org/terasology/rendering/nui/ScaleMode.java

```diff
@@ -82,6 +82,29 @@ public Vector2f scaleForRegion(Rect2i region, int actualWidth, int actualHeight)
         public String toString() {
             return "tiled";
         }
+    },
+    Automatic_selector {
+        @Override
+        public Vector2f scaleForRegion(Rect2i region, int actualWidth, int actualHeight) {
+
+                if(actualHeight < 500)
```

Comment:
This really shouldn't work off of a hardcoded size. Why 500? It seems really arbitrary. Should we treat an image that's 700x700 the same as one that's 600x3?